src/components/Table/Table.tsx

```diff
@@ -117,7 +117,7 @@ export interface TableProps<I> extends QAProps {
      *
      * @returns {DescriptorType} {@link DescriptorType}
      */
-    getRowDescriptor?: (item: I, index: number) => DescriptorType;
+    getRowDescriptor?: (item: I, index: number) => DescriptorType | undefined;
```

Comment:
Why? If `getRowDescriptor` is passed, it should definitely return something.

Reply:
`it should definitely return something` - it will be more convenient  in case

```
const getRowDescriptor = (item) => {
    if (isValidRow(item)) {
        return {...}
    }
    
    // otherwise i do not want even return empty object
}
```